src/main/java/com/blockout/controls/Image.java

```diff
@@ -97,7 +97,7 @@ private void loadMapDimensions()
             }
             catch (IOException e)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1166)